### PR TITLE
:sparkles: feat: pool header

### PR DESCRIPTION
**Pull request by atilasousa**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206685046825512

### CODE REVIEW

File: views/pools/header/action-group.tsx [hunk 1]
```diff
@@ -3,75 +3,75 @@ import { useRouter } from 'next/router';
...
           }
         >
-          Find Pool
+          Create pool
         </Button>
       </Box>
       <Box display={['flex', 'flex', 'flex', 'none']} gap="xs">
         <Button
           isIcon
+          color="black"
```
Comment (MarioBatalha) [inline views/pools/header/action-group.tsx:61]: use value from theme.

File: views/pools/header/search-mobile.tsx [hunk 2]
```diff
@@ -22,16 +22,24 @@ const SearchMobile: FC<SearchMobileProps> = ({
...
         }
+        fieldProps={{
+          borderRadius: 'xs',
+        }}
         placeholder="Search"
       />
       <Button
         isIcon
         my="auto"
+        color="black"
```
Comment (MarioBatalha) [inline views/pools/header/search-mobile.tsx:33]: Here too

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

**Review by git-marcopitra — APPROVED** [review]

File: views/pools/index.tsx [hunk 3]
```diff
@@ -16,19 +16,17 @@ const Pools: FC = () => {
...
   return (
-    <Layout title="Pools">
-      <Box
-        p={['s', 's', 's', 'l']}
-        borderRadius="xs"
-        bg="container"
-        mx={['0', '0', '0', '0', '9xl']}
-      >
-        <Header setTab={setTab} currentTab={tab} />
+    <Layout title="">
```
Comment (KipandaJr) [inline views/pools/index.tsx:19]: u can remove this props `title`

File: views/pools/index.tsx [hunk 4]
```diff
@@ -16,19 +16,17 @@ const Pools: FC = () => {
...
+      <Box mx={['0', '0', '0', '0', '9xl']}>
+        <Box py="1.5rem">
```
Comment (KipandaJr) [inline views/pools/index.tsx:21]: check this too

**Review by KipandaJr — CHANGES_REQUESTED** [review]

**Review by KipandaJr — APPROVED** [review]

**Review by MarioBatalha — APPROVED** [review]

**Pull request merged**